.github/workflows/contributor_response.yml

```diff
@@ -0,0 +1,41 @@
+name: Auto-response to Outside Contributors
```

Comment:
(recommend, non-blocking) One place `name` is used is in the bar [listing all the workflows](https://github.com/civiform/civiform/actions). Since this bar can end up truncating the text I recommend making `name` slightly shorter.

(recommend, non-blocking) I also recommend setting the `run-name` this value can be longer and is shown in the details of the running job. If set it replace `name` in many places.

Reply:
I didn't know about `run-name`.  Thanks!  I've shortened the name and added a `run-name`.

Reply:
Awesome! Most of our older ones don't use them because people didn't know about it.